lua/terrortown/lang/de.lua

```diff
@@ -466,7 +466,7 @@ L.round_post = "Runde vorbei"
 
 -- Health, ammo and time area
 L.overtime = "VERLÄNGERUNG"
-L.hastemode = "HAST MODUS"
+L.hastemode = "HASTE MODUS"
```

Comment:
Aber "Haste" heißt "Hast", oder "Eile". Wieso änderst du das?

Reply:
Kennst du das, wenn du ein Wort liest, und es einfach falsch verstehst? Ich habe die ganze Zeit "Hast Modus" (hast wie in haben) gelesen....... Das hat in meinem Kopf dann keinen Sinn gemacht xD

Aber generell.. Ist Hast Modus wirklich ein gutes Wort dafür?
Wenn ja, dann ändere ich es wieder zurück.

Reply:
Ich fände Eilmodus (ein Wort) auch ganz nett. Aber Hast Modus "kennt man" halt. Das würde dafür sprechen.

Und ja, das kenne ich. Brathering!